### ISSUE TITLE
nodeping.com

### ISSUE THREAD

**Issue by cscheid** [ieee-vgtc/ieeevis.org#358]
I had a trial account at nodeping.com to run liveness checks. That was a good thing, but it costs $15 a month for the full version. We should consider something like that.

**Comment by yelper** [comment]
https://updown.io/#pricing might be an option.  Seems like it can get as cheap as $.10 a month.

**Comment by cscheid** [comment]
Yup, that's what I'm using now :) I added ieeevis.org to my personal account, so we're good for now.
